src/avkit.cs

```diff
@@ -647,11 +647,6 @@ interface AVRoutePickerView
 #endif
 		[NullAllowed, Export ("delegate", ArgumentSemantic.Weak)]
 		IAVRoutePickerViewDelegate Delegate { get; set; }
-
-		[NullAllowed, Export ("delegate", ArgumentSemantic.Weak)]
-		[Sealed]
-		NSObject WeakDelegate { get; set; }
-		
```

Comment:
Not sure that's correct. It's the `Delegate` that should be using `[Wrap]` over the `WeakDelegate`

Reply:
Yes, before the changes I had this (not for Mac) : 
```C#
[NullAllowed, Export ("delegate", ArgumentSemantic.Weak)]
IAVRoutePickerViewDelegate Delegate { get; set; }
```
But I had to add it for Mac, sharpie gave me this : 
```C#
[Wrap ("WeakDelegate")]
[NullAllowed]
IAVRoutePickerViewDelegate Delegate { get; set; }

[NullAllowed, Export ("delegate", ArgumentSemantic.Weak)]
NSObject WeakDelegate { get; set; }
```

But this way I change it for iOS too. Maybe I should just put the weak delegate only for Mac ? Like this : 

```C#
#if !MONOMAC
[NullAllowed, Export ("delegate", ArgumentSemantic.Weak)]
IAVRoutePickerViewDelegate Delegate { get; set; }
#else
[Wrap ("WeakDelegate")]
[NullAllowed]
IAVRoutePickerViewDelegate Delegate { get; set; }

[NullAllowed, Export ("delegate", ArgumentSemantic.Weak)]
NSObject WeakDelegate { get; set; }
#endif
```

As I understood the use of ```Wrap``` allow us to wrap a weakly-typed object with a strongly-typed object. So It's "better" but I don't have to change the way It was for iOS.

Reply:
For a _short_ while we did not provide `WeakDelegate` properties but that decision was reversed :) We have not gone thru everything, but we're _fixing_ (adding) them as we update.

If done correctly it's not a breaking change (just verify the API diff to be sure).